qt/python/mantidqt/widgets/codeeditor/interpreter.py

```diff
@@ -362,8 +364,3 @@ def _finish(self, success, task_result):
 
     def _create_status_msg(self, status, timestamp, elapsed_time):
         return IDLE_STATUS_MSG + ' ' + LAST_JOB_MSG_TEMPLATE.format(status, timestamp, elapsed_time)
-
-    def _on_progress_update(self, lineno):
-        """Update progress on the view taking into account if a selection of code is
```

Comment:
Thanks for removing this. We missed it when this was changed just before code freeze.